src/components/operations/operation-details/ko/runtime/operation-console.html

```diff
@@ -244,72 +244,17 @@ <h3>HTTP request</h3>
 
 <!-- ko if: responseStatusCode -->
 <div class="panel panel-dark menu menu-horizontal animation-fade-in" data-bind="scrollintoview: {}">
-    <ul class="nav">
-        <li class="nav-item" data-bind="css: { 'nav-item-active': $component.responseActiveTab() === 'message' }">
-            <a href="#" class="nav-link"
-                data-bind="click: $component.selectMessageTab">Message</a>
-        </li>
-        <li class="nav-item" data-bind="css: { 'nav-item-active': $component.responseActiveTab() === 'trace' }">
-            <a href="#" class="nav-link"
-                data-bind="click: $component.selectTraceTab">Trace</a>
-        </li>
-    </ul>
+    <h3>Message</h3>
```

Comment:
Rupeng we don't need it anymore `<h3>Message</h3>`, there is a heading HTTP response, that should be enough.

Reply:
ok!